Fix/dispatches

iparask opened the pull request:
Makes sure a single workflow is triggered by each resource and that the badges are on the main README

mtitov commented on README.md:
```diff
@@ -31,3 +31,10 @@ supported backends, please refer to the documentation.
+## Integration Tests status
+These badges show the state of the current integration tests on different HPCs RADICAL Pilot supports
+
+[![ORNL Summit Integration Tests](https://github.com/iparask/git_actions/actions/workflows/summit.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/summit.yml)
+[![PSC Bridges2 Integration Tests](https://github.com/iparask/git_actions/actions/workflows/bridges.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/bridges.yml)
+[![SDSC Comet Integration Tests](https://github.com/iparask/git_actions/actions/workflows/comet.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/comet.yml)
```
wrong paths for badges?

iparask commented:
I updated the badges, the event triggers and added more information in the integration test readme file.

I changed the event names and have them unique per resource. I found that all three workflows were triggered by the same event name.

andre-merzky reviewed (approved):
LGTM, thanks!

mtitov commented on README.md:
```diff
@@ -31,3 +31,10 @@ supported backends, please refer to the documentation.
+## Integration Tests status
+These badges show the state of the current integration tests on different HPCs RADICAL Pilot supports
+
+[![ORNL Summit Integration Tests](https://github.com/iparask/radical-cybertools/radical.pilot/actions/workflows/summit.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/summit.yml)
+[![PSC Bridges2 Integration Tests](https://github.com/iparask/radical-cybertools/radical.pilot/actions/workflows/bridges.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/bridges.yml)
+[![SDSC Comet Integration Tests](https://github.com/iparask/radical-cybertools/radical.pilot/actions/workflows/comet.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/comet.yml)
```
@iparask sorry, your name is still in urls for badges (`iparask`)

mtitov reviewed (approved):
LGTM!
